web/src/testgrid-grid-cell.ts

```diff
@@ -0,0 +1,68 @@
+import { LitElement, html, css } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { TestStatus } from "./gen/pb/test_status/test_status";
+
+@customElement('testgrid-grid-cell')
+export class TestgridGridCell extends LitElement{
+    // Styling for status attribute corresponds to test_status.proto enum.
+    static styles = css`
+    :host {
+      min-width: 80px;
+      width: 80px;
+      min-height: 22px;
+      max-height: 22px;
+      color: #000;
+      background-color: #ccc;
+      text-align: center;
+      font-family: Roboto, Verdana, sans-serif;
+      font-weight: bold;
+      display: flex;
+      justify-content: center;
+      align-content: center;
+      flex-direction: column;
+      box-sizing: border-box;
+      font-size: 12px;
+    }
+
+    :host([status="NO_RESULT"]) {
```

Comment:
may be assign different classes to statuses... this way we will define common css styles, import them where necessary and use across cell colors and tab summary colors

Reply:
Will consider for refactor, but there's a few things I want to fix more holistically in a follow-up after I check some more resources. Filed #1205 instead if that works!